### ISSUE TITLE
webpack compile error: Argument of type '{ title: TemplateRef<any>...' is not assignable to parameter of type 'ConfigInterface'.

### ISSUE THREAD

**Issue by tommas3210** [NG-ZORRO/ng-zorro-antd#989]
## Version

0.6.13

## Environment

ng-zorro 0.6.13

## Reproduction link

[https://stackblitz.com/edit/ng-zorro-antd-setup-wtzhns?file=app/app.component.ts](https://stackblitz.com/edit/ng-zorro-antd-setup-wtzhns?file=app/app.component.ts)

## Steps to reproduce

```typescript
import { Component, TemplateRef, ViewChild } from '@angular/core'
import { NzModalService } from 'ng-zorro-antd'

@Component({
  selector: 'my-app',
  template: `
    <button (click)="openModal()">Open</button>
    <ng-template #modalTitle>This is title</ng-template>
    <ng-template #modalBody>Hello World!!!</ng-template>
  `
})
export class AppComponent {

  @ViewChild('modalTitle') modalTitle: TemplateRef<any>;
  @ViewChild('modalBody') modalBody: TemplateRef<any>;

  openModal(): void {
    this.nzModalService.open({
      title: this.modalTitle,   // <==  compile error here!!
      content: this.modalBody
    });
  }
  constructor(private nzModalService: NzModalService) { }
}
```

## What is expected?

use templateRef

## What is actually happening?

```
Argument of type '{ title: TemplateRef<any>; content: TemplateRef<any>; }' is not assignable to parameter of type 'ConfigInterface'.
  Types of property 'title' are incompatible.
    Type 'TemplateRef<any>' is not assignable to type 'string'.
```

**Comment by vthinkxie** [comment]
will be fixed in next version.

**Comment by vthinkxie** [comment]
fixed in version 0.7.0
